.github/workflows/release-please.yml

```diff
@@ -54,7 +54,18 @@ jobs:
       - name: Run E2E tests
         run: npm run test:e2e
 
-      - name: Publish package
+      - name: Publish netlify-cli package
         run: npm publish --provenance
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+
+      - name: Prepare netlify package
+        run: node scripts/netlifyPackage.js prepare
+
+      - name: Publish netlify package
+        run: npm publish --provenance
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+
+      - name: Verify netlify package
+        run: node scripts/netlifyPackage.js verify
```

Comment:
suggestion: It seems to me that this is an e2e test, but it isn't quite framed that way, it can't be run locally against local changes nor in CI, and if it catches an issue someday it would be pretty late in the process (after it's public). Since we already have a nice e2e setup in this repo that does _precisely_ what we want here (using verdaccio), I'd suggest adding a test there and removing this!

Reply:
Agreed. Moved to the E2E setup in https://github.com/netlify/cli/pull/7293/commits/0bbc22dd1b3d398d66303a65003ed23970ce5c34.

Reply:
Ugh. It's failing on Windows somehow. I'll come back to this when I have some time.